build.sbt

```diff
@@ -49,7 +52,8 @@ val baseSettings = Seq(
     "org.typelevel" %% "cats-core" % catsVersion,
     "com.twitter" %% "finagle-http" % finagleVersion,
     scalaOrganization.value % "scala-reflect" % scalaVersion.value,
-    "io.catbird" %% "catbird-util" % catbirdVersion
+    "io.catbird" %% "catbird-util" % catbirdVersion,
```

Comment:
I'm trying to figure out if we still need a catbird dependency? I'd assume `finch-iteratee` would need it but it seems to depend on `iteratee-twitter` already. What happens if we drop it?

Reply:
yes I had the same idea, will see how it works, also removing added whitespaces